sdk/monitor/azure-monitor-opentelemetry/samples/tracing/azure_ai_inference.py

```diff
@@ -0,0 +1,39 @@
+from os import environ
+import os
+
+from azure.ai.inference import ChatCompletionsClient
+from azure.ai.inference.models import SystemMessage, UserMessage, CompletionsFinishReason
+from azure.core.credentials import AzureKeyCredential
+
+from azure.monitor.opentelemetry import configure_azure_monitor
+from opentelemetry import trace
+
+# Set up exporting to Azure Monitor
+configure_azure_monitor()
```

Comment:
The other sample has this:

  # There is no need to pass APPLICATIONINSIGHTS_CONNECTION_STRING explicitly,
    # as it is read from this environment variable by the configure_azure_monitor function
    # It's provided here for clarity.
    configure_azure_monitor(connection_string=os.environ["APPLICATIONINSIGHTS_CONNECTION_STRING"])

Can we use the same lines in all samples? I would just call `configure_azure_monitor()` and put a comment above that says this function reads the env variable APPLICATIONINSIGHTS_CONNECTION_STRING